wsd/COOLWSD.cpp

```diff
@@ -3724,22 +3699,23 @@ class PrisonerRequestDispatcher final : public WebSocketHandler
             LOG_TRC("Child connection with URI [" << COOLWSD::anonymizeUrl(request.getUrl())
                                                   << ']');
             Poco::URI requestURI(request.getUrl());
-#ifndef KIT_IN_PROCESS
-            if (requestURI.getPath() == FORKIT_URI)
-            {
-                if (socket->getPid() != COOLWSD::ForKitProcId)
+            if(!Util::isKitInProcess()) {
```

Comment:
Please use clang-format to have uniform formatting everywhere. You can configure vscode to apply formatting only on edited lines.